### PR TITLE
artifact-registry: Support cleanup policies

### PR DESCRIPTION
**Pull request by noony**
This PR aim to add support of cleanup policies for artifact registry module.

ie : https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/artifact_registry_repository#example-usage---artifact-registry-repository-cleanup

---
**Checklist**

I applicable, I acknowledge that I have:
- [x] Read the [contributing guide](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md)
- [x] Ran `terraform fmt` on all modified files
- [x] Regenerated the relevant README.md files using [`tools/tfdoc.py`](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/master/CONTRIBUTING.md#fabric-tools)
- [x] Made sure all relevant tests pass

### CODE REVIEW

File: modules/artifact-registry/README.md [hunk 1]
```diff
@@ -100,14 +101,50 @@ module "registry-maven" {
 
 # tftest modules=2 resources=2
 ```
+
+## Docker Cleanup Policies
```
Comment (juliocc) [inline modules/artifact-registry/README.md:105]: Change to just "Cleanup Policies" (cleanup policies apply to all formats except Apt and Yum)

**Review by juliocc — APPROVED** [review]
Very nice. Thanks for the PR

Can you fix the linting issues?

**Comment by wiktorn** [comment]
> Can you fix the linting issues?
In terms of linting issue, the message is a bit cryptic, but you need to just use alphabetic order of variables in `variables.tf`

File: modules/artifact-registry/README.md (hunk 1 above)
Comment (noony) [reply modules/artifact-registry/README.md]: fixed

**Comment by noony** [comment]
Fixes on readme and variables.tf lint done

**Pull request merged**